src/client/delphi_epidata.py

```diff
@@ -665,3 +665,44 @@ def covid_hosp_facility_lookup(
       raise Exception('one of `state`, `ccn`, `city`, `zip`, or `fips_code` is required')
     # Make the API call
     return Epidata._request(params)
+
+  # Fetch Delphi's COVID-19 Surveillance Streams
```

Comment:
No idea why we comment these twice, but they should both be correct

```suggestion
  # Fetch Delphi's COVID-19 Nowcast sensors
```